File: dart/isogram/lib/isogram.dart
class Isogram {
  bool isIsogram(String s) {
    var charsSeen = new Set<String>();
    var normalized = s.toLowerCase().replaceAll(RegExp('[ -]'), '');
    for (int i = 0; i < normalized.length; i++) {
      String char = normalized[i];
      if (charsSeen.contains(char)) return false;
      charsSeen.add(char);
    }
    return true;
  }
}

File: kotlin/isogram/src/main/kotlin/Isogram.kt
package Isogram

fun isIsogram(s: String): Boolean {
  var set = HashSet<Char>()
  for (c in s.toLowerCase()) {
    if (c.isLetter()) {
      if (set.contains(c)) {
        return false
      }
      set.add(c)
    }
  }
  return true
}